service/docs/source/runtime.rst

```diff
@@ -70,37 +76,73 @@ Runtime launch options.
          trace file per host, and one report across all hosts.
    :align: center
 
-   The geopmlaunch tool is the main user interface to the GEOPM Runtime. It
+
+.. admonition:: Quick start for MPI applications
+
+   The ``geopmlaunch`` tool is the recommended user interface to launch the GEOPM Runtime for 
+   profiling and optimizing MPI applications. It
```

Comment:
Minor nit: maybe review column limits, follow same limit as in the rest of the document.